### ISSUE TITLE
Test harness header should be accessible 

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#445]
Test harness header -
![test-harness-header](https://user-images.githubusercontent.com/25124000/104394746-1245fb00-54fc-11eb-8f0c-8ad87b621b27.png)

Add `aria-label` attribute to links in test harness header. This will fix following A11y violations - 
<img width="830" alt="Screen Shot 2021-01-12 at 5 32 20 PM" src="https://user-images.githubusercontent.com/25124000/104394786-2984e880-54fc-11eb-9d09-b14f9b2b2141.png">
